domains/muhalvin.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "For my portofolio and personal project",
+    "repo": "https://github.com/muhalvin/muhalvin.github.io",
+    "owner": {
+        "username": "muhalvin",
+        "email": "work.muhalvin@gmail.com  ",
+    },
+    "record": {
+        "CNAME": "muhalvin.github.io"
```

Comment:
Please add more stuff to your website

Reply:
done sir, i've updated my website just now.

Reply:
![image](https://github.com/is-a-dev/register/assets/97495721/d0411c68-860b-4fe4-b13b-a2bf896879cf)


muhalvin.github.io